packages/editor/__tests__/specs/userRoleEditPanel.tsx

```diff
@@ -211,21 +211,24 @@ describe('User Role Edit Panel', () => {
       permissionListQuery
     ]
 
-    const {asFragment, container} = render(
+    const {asFragment, getByTestId, getByLabelText} = render(
       <MockedProvider mocks={mocks} addTypename={false}>
         <UserRoleEditPanel />
       </MockedProvider>
     )
     await actWait()
     const initialRender = asFragment()
+    const nameInput = getByLabelText('userRoles.panels.name*')
+    const descriptionInput = getByLabelText('userRoles.panels.description')
+    const saveButton = getByTestId('saveButton')
 
-    fireEvent.change(container.querySelector('input[name="userRoles.panels.name"]')!, {
+    fireEvent.change(nameInput!, {
```

Comment:
forgot to tell you, please remove the non-null assertion - if we don't find the element on the page the tests should fail, we shouldn't enforce it like this :)

Reply:
react testing library does fail when it can't find the element. It's their typings that is wrong since they are probably not made for strict mode